content/graph.go

```diff
@@ -74,7 +74,7 @@ func Successors(ctx context.Context, fetcher Fetcher, node ocispec.Descriptor) (
 			return nil, err
 		}
 		return index.Manifests, nil
-	case artifactspec.MediaTypeArtifactManifest:
+	case artifactspec.MediaTypeArtifactManifest: // TODO: deprecate
```

Comment:
How about just removing this switch case?

Reply:
Will do it after refactoring `Pack` and `ExtendedCopy`.